src/test/java/org/springframework/data/redis/cache/RedisCacheTests.java

```diff
@@ -15,10 +15,10 @@
  */
 package org.springframework.data.redis.cache;
 
-import static org.assertj.core.api.Assertions.*;
```

Comment:
The import changes do not seem in line with the import order defined in `spring-data-build`.